src/client/components/Inputs/InputText/InputText.tsx

```diff
@@ -0,0 +1,25 @@
+import './InputText.scss'
```

Comment:
what do you thin naming them eg
```suggestion
import `'./InpuText.module.scss
```